### PR TITLE
Add mouse interaction to canvas

### PR DESCRIPTION
**Pull request by david-qz**
This PR enables users to rotate and zoom on the rendered model with their mouse.

### CODE REVIEW

**Review by Allison-Ause — COMMENTED** [review]
I appreciate the frankness of your assessment around the Euler angles. Good to leave room for improvement if there's time. This looks great!

**Review by Allison-Ause — COMMENTED** [review]
The changes you made to the WheelEvent are very interesting. Looks good.

File: client/components/scene/scene.tsx [hunk 1]
```diff
@@ -58,6 +58,14 @@ export default function Scene({ style }: Props): ReactElement {
...
+    if (!sceneRef.current) return;
+    const scene = sceneRef.current;
+
+    const distance = scene.cameraDistance;
+    scene.cameraDistance = distance + (distance / e.deltaY) * 20;
+  }
+
   return (
     <div className={styles.canvasContainer} style={style}>
       <canvas
```
Comment (Allison-Ause) [inline client/components/scene/scene.tsx:71]: What would a possible test for this look like on the Front-End? Checking to see if the WheelEvent response triggers as expected given certain circumstance?

File: client/components/scene/webgl/scene-renderer.ts [hunk 2]
```diff
@@ -110,4 +115,14 @@ export default class SceneRenderer {
     // Continue render loop if we're still running
```
Comment (Allison-Ause) [inline client/components/scene/webgl/scene-renderer.ts:117]: This is a very helpful comment!

**Review by Allison-Ause — APPROVED** [review]
LGTM!

File: client/components/scene/scene.tsx (hunk 1 above)
Comment (david-qz) [reply client/components/scene/scene.tsx:71]: I guess I would need to mock the SceneRenderer, trigger a WheelEvent, and check that the camera distance gets moved in the expected direction. I will leave that for the Future (TM).

**Pull request merged**